sprockets/mixins/correlation/__init__.py

```diff
@@ -1,5 +1,18 @@
-from .mixins import HandlerMixin
+try:
+    from .mixins import CorrelationAdapter, HandlerMixin, LoggingMixin
```

Comment:
Do all 3 of these come from the same origin package? ie is this all from ``sprockets.mixins.logging`` or something?